bigtable-dataflow-parent/bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java

```diff
@@ -1090,6 +1092,7 @@ public void finishBundle(Context context) throws Exception {
 
     // Stats
     private final Aggregator<Long, Long> mutationsCounter;
+    private volatile Map<String, BufferedMutator> mutators;
```

Comment:
Dataflow serializes DoFns, and anything that isn't serializable, or should be per bundle needs to be volatile.  BigtableBufferedMutators aren't serializable...